src/main/java/com/growup/pms/user/repository/UserRepository.java

```diff
@@ -11,6 +11,8 @@ public interface UserRepository extends JpaRepository<User, Long>, QueryDslUserR
 
     Optional<User> findByEmail(String email);
 
+    Boolean existsByProfileNickname(String nickname);
```

Comment:
```suggestion
    Boolean existsByNickname(String nickname);
```
`nickname` 은 저희 DB상 고유한 이름으로 알고 있기 때문에, 굳이 앞에 `Profile` 을 붙여야 할 필요는 없지 않을까 싶습니다.